### PR TITLE
Add user list

### PR DESCRIPTION
**Pull request by alavieille**
https://github.com/open-orchestra/open-orchestra/pull/1101
https://github.com/open-orchestra/open-orchestra-libs/pull/71
https://github.com/open-orchestra/open-orchestra-mongo-libs/pull/64
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/2013
https://github.com/open-orchestra/open-orchestra-user-bundle/pull/133

### CODE REVIEW

File: BackofficeBundle/Resources/translations/interface.fr.yml [hunk 1]
```diff
@@ -11,6 +11,8 @@ open_orchestra_backoffice:
         yes: Oui
         no: Non
         filter: Filtrer les résultats
+    no_item: Il n’y a pas encore d’éléments pour cette vue
```
Comment (ngilain) [inline BackofficeBundle/Resources/translations/interface.fr.yml]: d'élément

File: BackofficeBundle/Resources/translations/interface.en.yml [hunk 2]
```diff
@@ -11,6 +11,8 @@ open_orchestra_backoffice:
         yes: Yes
         no: No
         filter: Filter results
+    no_item: There are no items for this view
```
Comment (ngilain) [inline BackofficeBundle/Resources/translations/interface.en.yml]: item

File: UserAdminBundle/Controller/Api/UserController.php [hunk 3]
```diff
@@ -141,20 +163,29 @@ public function addGroupAction($groupId, $userId)
     }
 
     /**
-     * @param int $userId
+     * @param Request $request
      *
-     * @Config\Route("/{userId}/delete", name="open_orchestra_api_user_delete")
+     * @Config\Route("/delete/users", name="open_orchestra_api_user_delete_multiple")
```
Comment (ngilain) [inline UserAdminBundle/Controller/Api/UserController.php]: user/delete-multiple instead ?

File: UserAdminBundle/Controller/Api/UserController.php [hunk 4]
```diff
@@ -141,20 +163,29 @@ public function addGroupAction($groupId, $userId)
...
+        $format = $request->get('_format', 'json');
+
+        $facade = $this->get('jms_serializer')->deserialize(
+            $request->getContent(),
+            $this->getParameter('open_orchestra_user_admin.facade.user_collection.class'),
+            $format
+        );
+        //$this->dispatchEvent(UserEvents::USER_DELETE, new UserEvent($user));
+
+        /*$user = $this->get('open_orchestra_user.repository.user')->find($userId);
```
Comment (ngilain) [inline UserAdminBundle/Controller/Api/UserController.php]: And here too

File: UserAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/User/UsersView.js [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+            filters[data.name] = data.value;
+        }
+        this._listView.filter(filters);
+
+        return false;
+    }
+
+    _remove() {
+        let users = this._collection.where({'delete': true});
+        console.log(users);
```
Comment (ngilain) [inline UserAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/User/UsersView.js]: log to remove

File: UserAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/User/UsersView.js [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+        this._collection.destroyModels(users);
+        console.log(this._collection);
```
Comment (ngilain) [inline UserAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/User/UsersView.js]: log to remove

File: UserAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/User/UserListView.js [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+     * @param {Object} td
+     * @param {Object} cellData
+     * @param {Object} rowData
+     *
+     * @private
+     */
+    _createCheckbox(td, cellData, rowData) {
+        let id = 'checkbox' + rowData.cid;
+        let $checkbox = $('<input>', {type: 'checkbox', id: id});
+        $checkbox.on('change', (event) => {
```
Comment (itkg-nanne) [inline UserAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/User/UserListView.js]: Could you use BackBoneView events?

File: UserAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/User/UsersView.js [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+    initialize({collection, settings}) {
+        this._collection = collection;
+        this._settings = settings;
+    }
+
+    /**
+     * Render nodes view
+     */
+    render() {
+        if (0 === this._collection.recordsTotal) {
```
Comment (itkg-nanne) [inline UserAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/User/UsersView.js]: Could it be done by a super.render(title) and a generic template?

File: UserAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/User/UsersView.js (hunk 5 above)
Comment (itkg-nanne) [inline UserAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/User/UsersView.js]: oups

File: UserAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/User/UsersView.js (hunk 6 above)
Comment (itkg-nanne) [inline UserAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/User/UsersView.js]: oups

**Comment by itkg-nanne** [comment]
:+1:

**Pull request merged**